src/CMakeLists.txt

```diff
@@ -450,6 +451,26 @@ IF(OpenMVG_USE_ALEMBIC)
 ENDIF(OpenMVG_USE_ALEMBIC)
 
 # ==============================================================================
+# FBX
+# ==============================================================================
+# - optional, builds SfM->FBX point cloud converter
+# ==============================================================================
+IF(OpenMVG_FBX_DIR)
```

Comment:
I agree. I suggest that we postpone merging this PR indefinitely, until we have a use for FBX format. After talking directly with NRK, It turned out that they can use Alembic and that they're much more flexible than we thought.